### PR TITLE
bitwise:  add page

### PR DESCRIPTION
**Pull request by deep5050**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/bitwise.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# bitwise
+
+> Multi base interactive calculator supporting dynamic base conversion and bit manipulation.
+> More information: <https://github.com/mellowcandle/bitwise>.
+
+- Interactive Mode:
```
Comment (owenvoke) [inline pages/linux/bitwise.md:6]:
```suggestion
- Run using interactive mode:
```

File: pages/linux/bitwise.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`bitwise`
+
+- Conversion From Decimal:
+
+`bitwise 12345`
```
Comment (owenvoke) [inline pages/linux/bitwise.md:12]: Just needs token syntax here, and a tweak to use sentence case:

```suggestion
- Convert from decimal:

`bitwise {{12345}}`
```

File: pages/linux/bitwise.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Conversion From Hex:
+
+`bitwise 0x563d`
```
Comment (owenvoke) [inline pages/linux/bitwise.md:16]: Just needs token syntax here, and a tweak to use sentence case:

```suggestion
- Convert from hexadecimal:

`bitwise {{0x563d}}`
```

File: pages/linux/bitwise.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- C-style Calculation:
```
Comment (owenvoke) [inline pages/linux/bitwise.md:18]:
```suggestion
- Convert a C-style calculation:
```

**Review by owenvoke — CHANGES_REQUESTED** [review]
Thanks @deep5050, just a few minor tweaks but it otherwise looks great. 👍🏻

**Review by owenvoke — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Looks ok to me! Thanks, @deep5050 :D

Note that you still need to sign the CLA before we can merge this. Simply click on the link i in the comment left by @CLAassistant above :-)

**Comment by deep5050** [comment]
@sbrl done :)

**Pull request merged**